.studio/automate-gateway

```diff
@@ -23,7 +23,7 @@ function gateway_integration() {
   # the relevant services up and running:
```

Comment:
In .studio/license-control-service, need the same fix on this line:
https://github.com/chef/automate/blob/cba671834c2277e6dd978b957552c48da009cb75/.studio/license-control-service#L23